pages/getting-started/writing-your-first-test.md

```diff
@@ -0,0 +1,92 @@
+# Escrevendo o primeiro teste
+```
+O que você vai aprender
+
+- Como começar a testar um novo projeto com Cypress
+- Como são reportados testes com sucesso e falhas
+- Testando a navegação web, buscando por elementos no DOM e escrevendo asserções
+```
+
+[Vídeo de exemplo.](https://vimeo.com/237115455)
+
+# Adicione um arquivo de teste
+
+Assumindo que você tenha [instalado](https://docs.cypress.io/guides/getting-started/installing-cypress.html#Installing) e [aberto](https://docs.cypress.io/guides/getting-started/installing-cypress.html#Opening-Cypress) o Cypress Test Runner, chegou a hora de escrever o primeiro teste. Vamos então:
+
+1. Criar um arquivo `sample_spec.js`.
+2. Checar que o Cypress atualizou a lista de arquivos de teste.
+3. Executar o Cypress Test Runner.
+
+Vamos criar um novo arquivo na pasta `cypress/integration`, a qual já foi criada:
+
+```touch {your_project}/cypress/integration/sample_spec.js```
+
+Uma vez que o arquivo tenha sido criado, o Cypress Test Runner deve exibi-lo na lista de testes de integração. O Cypress monitora mudanças em seus arquivos de teste e os exibe imediatamente.
+
+Mesmo que ainda não tenha escrito nenhum teste - tudo bem - clique em `sample_spec.js` e veja o Cypress iniciar o navegador.
+
+> O Cypress abre o teste em um navegador previamente instalado em seu sistema operacional. Você pode ler mais sobre como isso é feito em Iniciando Navegadores.
+
+[Vídeo de exemplo](https://docs.cypress.io/img/snippets/empty-file-30fps.mp4)
+
+Agora estamos oficialmente no [Test Runner](https://docs.cypress.io/guides/core-concepts/test-runner.html). É nele onde vamos gastar a maioria do tempo testando.
+
+> Note que o Cypress exibe a mensagem que não pode encontrar nenhum teste. Isso é normal - testes ainda não foram escritos! Você verá essa mensagem também caso exista algum erro ao analisar os arquivos de teste. Você pode abrir o Dev Tools para inspecionar o console e buscar por erros de sintaxe que impediu o Cypress de ler seus arquivos de testes.
+
+# Escreva seu primeiro teste
+
+Chegou a hora de escrever o nosso primeiro teste. Vamos então:
+
+1. Escrever nosso primeiro teste com sucesso.
+2. Escrever nosso primeiro teste com falha.
+3. Ver o Cypress recarregar o teste em tempo real.
+
+A medida que salvamos nosso novo teste, veremos que o navegador recarrega automaticamente em tempo real.
+
+Abra sua IDE favorita e adicione o código abaixo no arquivo de teste `sample_spec.js`.
+
+```
+describe('My First Test', () => {
+  it('Does not do much!', () => {
+    expect(true).to.equal(true)
+  })
+})
+```
+
+Quando salvar esse arquivo você deve ver o navegador recarregar.
+
+Embora não faça nada, esse é nosso primeiro teste com sucesso!
+
+No [log de comandos](https://docs.cypress.io/guides/core-concepts/test-runner.html#Command-Log) você vai ver que o Cypress exibe a suíte de teste, o teste e a primeira asserção (que deve estar em verde - sucesso).
```

Comment:
We need to link for our translated content

Reply:
Makes sense :) I'll update all the places. What about the ones not translated yet.. should I remove the links or keep the English ones until we have them translated?

Reply:
@lucastfa-toptal you can leave it like this example:

```
[//]: <> (TODO - Adicionar link CLI)

Você pode [ler mais sobre CLI aqui.](#maissobrecli)
```

Concerns?

Reply:
Sounds good, but I prefer to add the TODO but also keep the english version for now, so we can track what page it is referencing, until it is translated.